quickstart/aspnet-core/README.md

```diff
@@ -0,0 +1,56 @@
+---
+page_type: sample
+languages:
+- csharp
+name: 'Quickstart: Use Azure Cache for Redis with an ASP.NET Core web app'
+description: Learn how to use an ASP.NET Core web application to connect to Azure Cache for Redis to store and retrieve data from the cache.
+products:
+- azure
+- aspnet-core
+- azure-cache-redis
+---
+# Quickstart: Use Azure Cache for Redis with an ASP.NET Core web app
+
+This sample shows you how to use an ASP.NET Core web application to connect to Azure Cache for Redis to store and retrieve data from the cache. See the [accompanying article](https://docs.microsoft.com/azure/azure-cache-for-redis/cache-web-app-aspnet-core-howto) on the documentation site for details, including best practices and how to create the sample code from scratch.
```

Comment:
Note that the accompanying article on docs.microsoft.com does not yet exist. I'll add a link for a corresponding PR for that soon.